ClLike/cl_like/bacco.py

```diff
@@ -117,22 +121,25 @@ def update_pk(self, cosmo, bcmpar=None, **kwargs):
         cospar = {
             'omega_cold': cosmo['Omega_c'] + cosmo['Omega_b'],
             'omega_baryon': cosmo['Omega_b'],
-            'sigma8_cold': cosmo.sigma8(),
             'ns': cosmo['n_s'],
             'hubble': h,
             'neutrino_mass': np.sum(cosmo['m_nu']),
             'w0': cosmo['w0'],
             'wa': cosmo['wa']}
+        if np.isnan(cosmo['A_s']):
+            cospar['sigma8_cold'] = cosmo.sigma8() # in this case sigma8 is being varied, here it is interpreted as sigma8_cold
+        else:
+            cospar['A_s'] = cosmo['A_s']
+        
         k_for_bacco = self.ks/h
         self.mask_ks_for_bacco = np.squeeze(np.where(k_for_bacco <= 0.75))
         k_for_bacco = k_for_bacco[self.mask_ks_for_bacco]
         if self.ignore_lbias:
             self.pk_temp = None
         else:
             self.pk_temp = np.array([self.lbias.get_nonlinear_pnn(k=k_for_bacco,
-                                                                allow_high_k_extrapolation=False, #this is bad, should be improved
```

Comment:
@matteozennaro could you remind me why this is not necessary anymore?

Reply:
Actually False is already the default so this doesn't change. In any case the extrapolation is now done here instead of baccoemu and this is better so I'm thinking of removing that keyword in the new version of baccoemu, so I already removed it from here.

Reply:
Ok. Thanks! I'll start running chains then.